[Merged by Bors] - chore: `n+1` to `n + 1`

adomani opened the pull request:
---
<!-- The text above the `---` will become the commit message when your
PR is merged. Please leave a blank newline before the `---`, otherwise
GitHub will format the text above it as a title.

For details on the "pull request lifecycle" in mathlib, please see:
https://leanprover-community.github.io/contribute/index.html

In particular, note that most reviewers will only notice your PR
if it passes the continuous integration checks.
Please ask for help on https://leanprover.zulipchat.com if needed.

To indicate co-authors, include lines at the bottom of the commit message
(that is, before the `---`) using the following format:

Co-authored-by: Author Name <author@email.com>

If you are moving or deleting declarations, please include these lines at the bottom of the commit message
(that is, before the `---`) using the following format:

Moves:
- Vector.* -> List.Vector.*
- ...

Deletions:
- Nat.bit1_add_bit1
- ...

Any other comments you want to keep out of the PR commit should go
below the `---`, and placed outside this HTML comment, or else they
will be invisible to reviewers.

If this PR depends on other PRs, please list them below this comment,
using the following format:
- [ ] depends on: #abc [optional extra text]
- [ ] depends on: #xyz [optional extra text]

-->

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)

bryangingechen commented on Mathlib/Algebra/LinearRecurrence.lean:
```diff
@@ -10,7 +10,7 @@ import Mathlib.LinearAlgebra.Dimension.Constructions
 
 Informally, a "linear recurrence" is an assertion of the form
-`∀ n : ℕ, u (n + d) = a 0 * u n + a 1 * u (n+1) + ... + a (d-1) * u (n+d-1)`,
+`∀ n : ℕ, u (n + d) = a 0 * u n + a 1 * u (n + 1) + ... + a (d-1) * u (n+d-1)`,
```
The spacing here is a bit inconsistent as well.

grunweg commented on Mathlib/Analysis/Calculus/ContDiff/Basic.lean:
```diff
@@ -1146,9 +1146,9 @@ section bundled
 /-- One direction of `contDiffWithinAt_succ_iff_hasFDerivWithinAt`, but where all derivatives are
 taken within the same set. Version for partial derivatives / functions with parameters. If `f x` is
-a `C^n+1` family of functions and `g x` is a `C^n` family of points, then the derivative of `f x` at
+a `C^n + 1` family of functions and `g x` is a `C^n` family of points, then the derivative of `f x` at
```
I personally wouldn't mind leaving the doc-string alone. (Are these also linted?)

adomani replied:
The doc-strings are not linted, but, for consistency, I have a slight preference for fixing them.  I won't be doing this systematically, but when I am using `sed`, I might as well include them!

adomani replied:
I agree with this dual policy, however, here I think that the better fix is the one that Bryan suggested, i.e. adding the braces, as well as the spaces!  :smile:

grunweg commented on Mathlib/Analysis/Calculus/ContDiff/Basic.lean:
```diff
@@ -1146,9 +1146,9 @@ section bundled
+a `C^n + 1` family of functions and `g x` is a `C^n` family of points, then the derivative of `f x` at
 `g x` depends in a `C^n` way on `x`. We give a general version of this fact relative to sets which
-may not have unique derivatives, in the following form.  If `f : E × F → G` is `C^n+1` at
+may not have unique derivatives, in the following form.  If `f : E × F → G` is `C^n + 1` at
```
Ditto

grunweg commented on Mathlib/Analysis/SpecialFunctions/Complex/LogBounds.lean:
```diff
@@ -216,7 +216,7 @@ lemma norm_log_one_add_half_le_self {z : ℂ} (hz : ‖z‖ ≤ 1 / 2) : ‖log
   linarith
 
-/-- The difference of `log (1-z)⁻¹` and its `(n+1)`st Taylor polynomial can be bounded in
+/-- The difference of `log (1-z)⁻¹` and its `(n + 1)`st Taylor polynomial can be bounded in
```
(`1-z`, that is)

adomani commented:
Ok, I'll finish up when I am back!

adomani commented:
I have fixed the long lines, added the braces above and also replaced `1-z`.  The `1-z` was a little disappointing, in that there were very few instances.  I think that I would rather have this PR merged and find another big batch while looking at the linter PR, if you agree.

bryangingechen commented on Mathlib/Computability/TMConfig.lean:
```diff
@@ -39,7 +39,7 @@ output, with enough expressivity to write any partial recursive function. The pr
 * `succ` returns the successor of the head of the input, defaulting to zero if there is no head:
   * `succ [] = [1]`
-  * `succ (n :: v) = [n + 1]`
+  * `succ (n :: v) = [n+1]`
```
Why are these spaces removed? (Same question below)

grunweg commented on Mathlib/Data/Fin/Fin2.lean:
```diff
@@ -35,9 +35,9 @@ universe u
 /-- An alternate definition of `Fin n` defined as an inductive type instead of a subtype of `ℕ`. -/
 inductive Fin2 : ℕ → Type
-  /-- `0` as a member of `Fin (n + 1)` (`Fin 0` is empty) -/
+  /-- `0` as a member of `Fin (n+1)` (`Fin 0` is empty) -/
```
You missed one here

adomani replied:
Fixed, thanks!

grunweg commented on Mathlib/Data/Fin/Fin2.lean:
```diff
@@ -35,9 +35,9 @@ universe u
+  /-- `0` as a member of `Fin (n+1)` (`Fin 0` is empty) -/
   | fz {n} : Fin2 (n + 1)
-  /-- `n` as a member of `Fin (n + 1)` -/
+  /-- `n` as a member of `Fin (n+1)` -/
```
and here

adomani replied:
Fixed as well!

grunweg commented on Mathlib/Analysis/Seminorm.lean:
```diff
@@ -1194,8 +1194,8 @@ lemma rescale_to_shell_zpow (p : Seminorm 𝕜 E) {c : 𝕜} (hc : 1 < ‖c‖)
   have cpos : 0 < ‖c‖ := by positivity
-  have cnpos : 0 < ‖c^(n+1)‖ := by rw [norm_zpow]; exact xεpos.trans hn.2
-  refine ⟨-(n+1), ?_, ?_, ?_, ?_⟩
+  have cnpos : 0 < ‖c^(n + 1)‖ := by rw [norm_zpow]; exact xεpos.trans hn.2
```
spaces around the ^? optional

adomani replied:
I'll leave this one: it will get picked up at some point and I'd rather get this over!

grunweg commented on Mathlib/AlgebraicTopology/SimplicialSet/KanComplex.lean:
```diff
@@ -39,7 +39,7 @@ for every nonzero `n : ℕ` and `0 ≤ i ≤ n`,
 lemma KanComplex.hornFilling {S : SSet.{u}} [KanComplex S]
     {n : ℕ} {i : Fin (n + 2)} (σ₀ : (Λ[n + 1, i] : SSet) ⟶ S) :
-    ∃ σ : Δ[n + 1] ⟶ S, σ₀ = Λ[n + 1, i].ι ≫ σ := by
+    ∃ σ : Δ[n+1] ⟶ S, σ₀ = Λ[n + 1, i].ι ≫ σ := by
```
This is inconsistent with the change in SimplicialSet/Horn.lean.

adomani replied:
Reverted, thanks!

grunweg commented on Mathlib/AlgebraicTopology/SimplexCategory/Basic.lean:
```diff
@@ -152,7 +152,7 @@ def mkOfLeComp {n} (i j k : Fin (n + 1)) (h₁ : i ≤ j) (h₂ : j ≤ k) :
   }
 
-/-- The "inert" morphism associated to a subinterval `j ≤ i ≤ j + l` of `Fin (n + 1)`. -/
+/-- The "inert" morphism associated to a subinterval `j ≤ i ≤ j + l` of `Fin (n+1)`. -/
```
Intended?

adomani replied:
No, thanks: reverted!

grunweg commented on Mathlib/AlgebraicTopology/ExtraDegeneracy.lean:
```diff
@@ -16,7 +16,7 @@ import Mathlib.Tactic.FinCases
 of a simplicial set `X` are contractible, it suffices to construct an extra
 degeneracy as it is defined in *Simplicial Homotopy Theory* by Goerss-Jardine p. 190.
-It consists of a series of maps `π₀ X → X _⦋0⦌` and `X _⦋n⦌ → X _⦋n+1⦌` which
+It consists of a series of maps `π₀ X → X _⦋0⦌` and `X _⦋n⦌ → X _⦋n + 1⦌` which
```
intended?

adomani replied:
No, reverted, thanks!

j-loreaux reviewed (commented):
bors d+

adomani reviewed (commented):
Thank you all for your comments!  I'll merge once the PR passes CI!

bryangingechen commented on Mathlib/Data/Nat/Digits/Defs.lean:
```diff
@@ -34,7 +34,7 @@ variable {n : ℕ}
 def digitsAux0 : ℕ → List ℕ
   | 0 => []
-  | n + 1 => [n + 1]
+  | n + 1 => [n+1]
```
Just FYI, there are still examples of  `[n+1]` here and below.

adomani replied:
Thanks!  This PR started in the wrong way: I should not have used `sed` indiscriminately to begin with.  I'll prepare the future PRs with better tools!
